tools/apksize/src/storage/storage.gradle

```diff
@@ -0,0 +1,35 @@
+// Copyright 2018 Google LLC
```

Comment:
Can we add a bug on Github to track that we don't have a sizing test for Firebase Inappmessaging Display SDK that we shipped yesterday?

Reply:
If we assign that to the Fiam team, it might serve as a way to test our documentation too 😄